api-docs/api-reference/methods/users/post-forgot-pwd-v2.0.rst

```diff
@@ -0,0 +1,92 @@
+.. _post-forgot-pwd-v2.0:
+
+Forgot Password
+~~~~~~~~~~~~~~~
+
+.. code::
+
+    POST /v2.0/users/RAX-AUTH/forgot-pwd
+
+Use the Forgot Password operation to receive an email with a password reset token.
+
+Identity users can ONLY use the token generated from the Forgot Password operation
+to reset their password via the :ref:`Reset Password <reset-password-v2.0>` operation.
+
```

Comment:
After you receive the password reset token, use the new token to submit a :ref:`Reset password <reset-password-v2.>` request. You can only use this token to reset your password.

.. note::
    The reset token has a limited lifespan.  If the reset operation fails, submit another forgot password request to 
    generate a new token.